cla_public/static-src/javascripts/modules/address-finder.js

```diff
@@ -134,6 +134,11 @@
         .before($('<div class="form-row field-error"><p>' + msg + '</p></div>'));
     },
 
+    showHelp: function (msg) {
+      this.$formGroup.find('.form-row')
```

Comment:
You need to clear it before adding. Otherwise:

![screen shot 2015-05-28 at 17 29 34](https://cloud.githubusercontent.com/assets/203886/7865245/36532916-055f-11e5-95a7-3a27f879cb21.png)